src/hooks/useGlobalAppState.ts

```diff
@@ -32,6 +36,10 @@ const handleWakeUp = async (
   }
 
   await handleSecurityFlowWakeUp(client)
+
+  // On appStart the homeview will hide the splashscreen
```

Comment:
even if this is not the "home cozy app" displayed inside the `homeview`?

Reply:
@acezard I think that I already answered this question during a visio call with @Crash-- 

For now HomeView only render the cozy-home app. And if, in the future, we plan to replace this in some context, then calling `hideSplashScreen()` should be part of the contract.

However if I remember correctly, @Crash-- was thinking about default cozy-app. Then here we display them in the CozyAppScreen. In that case, the HomeView will be still displayed in the background.